--- afl-as.c.orig	2013-06-21 23:25:38.000000000 +0200
+++ afl-as.c	2013-06-22 06:08:59.000000000 +0200
@@ -65,8 +65,8 @@
   as_params[0] = "as";
   as_params[argc] = 0;
 
-  if (!strcmp(as_params[1], "--64"))
-    FATAL("64-bit compilation not supported yet (use gcc -m32)");
+  if (strcmp(as_params[1], "--64"))
+    FATAL("32-bit compilation not supported");
 
   input_file = as_params[argc - 1];
 
